src/components/panels/collapsible-panel/collapsible-panel.js

```diff
@@ -89,6 +89,7 @@ export default class CollapsiblePanel extends React.PureComponent {
             })}
           >
             <div
+              onClick={this.props.isDisabled ? undefined : toggle}
```

Comment:
This might result in bugs when there's a button/input in the header and the event propagation is not stopped. Our `headerControls` should be safe though, as we already call `event.stopPropagation()` for them.

Reply:
okay, I'll try adding a test then 👍

Reply:
ready for re-review